apps/extension/public/manifest.json

```diff
@@ -11,9 +11,16 @@
   "content_scripts": [
     {
       "matches": ["<all_urls>"],
-      "js": ["js/content-scripts.js"],
+      "js": ["js/injected-connection-manager.js"],
       "run_at": "document_start",
       "all_frames": true
+    },
+    {
+      "matches": ["<all_urls>"],
+      "js": ["js/expose-penumbra-global.js"],
```

Comment:
Can you double-check that the web app + extension features all still work? It looks like we are having trouble installing (or maybe it's my machine?)

<img width="503" alt="Screenshot 2023-12-06 at 11 36 38 AM" src="https://github.com/penumbra-zone/web/assets/16624263/42195b01-d497-4622-9ddf-d75520f0a668">

Would be nice to confirm that the new extension + web app, works end-to-end with the current feature set.